src/app/src/components/Submissions/Detail/Map.js

```diff
@@ -0,0 +1,78 @@
+import { Box, Button, Flex, HStack, Icon, Text } from '@chakra-ui/react';
+import { ChatAltIcon, DownloadIcon, MailIcon } from '@heroicons/react/outline';
+import { InformationCircleIcon } from '@heroicons/react/solid';
+import { MapContainer } from 'react-leaflet';
+
+import { DefaultBasemap } from '../../Layers/Basemaps';
+import MapPanes from '../../MapPanes';
+
+import { MAP_CENTER, MAP_INITIAL_ZOOM } from '../../../constants';
+
+export default function Map({ submission }) {
+    return (
+        <>
+            <MapContainer
+                center={MAP_CENTER}
+                zoom={MAP_INITIAL_ZOOM}
+                zoomControl={false}
+                style={{ height: '100%' }}
+            >
+                <MapPanes>
+                    <DefaultBasemap />
+                    <MapButtons submission={submission} />
+                    <SubmissionStatusBar submission={submission} />
+                </MapPanes>
+            </MapContainer>
+        </>
+    );
+}
+
+function MapButtons({ submission }) {
+    return (
+        <Box position='absolute' zIndex={1000} top={22} w='100%'>
+            <Flex justify='space-evenly'>
+                <MapButton icon={ChatAltIcon}>Review map</MapButton>
+                <MapButton icon={MailIcon}>Email</MapButton>
+                <MapButton icon={DownloadIcon}>Download</MapButton>
+            </Flex>
+        </Box>
+    );
+}
+
+function MapButton({ icon, children }) {
+    return (
+        <Button
+            size='lg'
+            variant='mapButton'
+            boxShadow='lg'
+            rightIcon={<Icon as={icon} />}
+        >
+            {children}
+        </Button>
+    );
+}
+
+function SubmissionStatusBar({ submission }) {
```

Comment:
My only concern about this is that we're covering up the ESRI attribution.

Reply:
Good catch. Fixed in [a59bcea](https://github.com/azavea/iow-boundary-tool/pull/104/commits/a59bcea22ceb00b3fa3ac1a0e960e1813b6b7707)

<img width="1232" alt="image" src="https://user-images.githubusercontent.com/1430060/194213214-7b6730a0-ee1f-458d-b492-0194468ba695.png">